content/blog/2022-02-09-predictkube-scaler.md

```diff
@@ -0,0 +1,158 @@
++++
+title = "Introducing PredictKube - an AI-based predictive autoscaler for KEDA made by Dysnix"
+date = 2022-02-09
```

Comment:
If we want to publish on Monday, let's update the date
```suggestion
date = 2022-02-14
```

Reply:
updated, thank you!